client/app/api/course/index.js

```diff
@@ -12,6 +13,7 @@ import ScribingQuestionAPI from './Assessment/question/scribing';
 const CourseAPI = {
   achievements: new AchievementsAPI(),
   assessment: AssessmentAPI,
+  categories: new CategoriesAPI(),
```

Comment:
would be better to nest this under assessment API, follow the controller structure.